### DISCUSSION TITLE
Tags: CVE Tags

### DISCUSSION THREAD

**Discussion by Neo23x0** [SigmaHQ/sigma#2146] [Ideas]
Currently the only tags that we use are MITRE ATT&CK techniques. 

What do you think about adding other types of tags as well? 

My first thought was to introduce CVE tags, e.g. 

```
tags:
    - cve.2021.12345
```

**Comment by jaegeral** [comment]
IMHO that could be super helpful to have not only in the references but dedicated as tags.

**Comment by frack113** [comment]
https://github.com/SigmaHQ/sigma-specification/blob/main/Tags_specification.md
